documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/sessions/session-architecture.adoc

```diff
@@ -13,18 +13,31 @@
 
 [[pg-server-session-architecture]]
 ==== Session Architecture
+==== Introduction
+Jetty session support has been architected to provide a core implementation that is independent of the servlet specification.
+This allows programmers who use core Jetty - without the servlet api - to still have classic servlet session-like support for their `Requests` and `Handlers`.
 
-Terminology::
-SessionIdManager::: is responsible for allocation of session ids
-HouseKeeper:::  is responsible for orchestrating the detection and removal of expired sessions
-SessionHandler::: is responsible for managing the lifecycle of sessions within its associated context
-SessionCache::: is an L1 cache of in-use `Session` objects
-Session::: is a stateful object representing a `HttpSession`
+Jetty also adapts these core classes to each of the various servlet specification environments to deliver classic `HttpSessions` for `Servlets`, `Filters` etc.
+
+We provide full support for the session lifecycle, in addition to L1 and L2 caching, and a number of options for persisting session data.
+These features are all available to both core and servlet environments.
+
+==== Terminology
+
+In order to set the scene for our discussion of sessions in Jetty, here are some of the most important concepts that we  will refer to throughout the documentation.
+
+SessionIdManager::: responsible for allocation of unique session ids. A `Server` can have at most 1 `SessionIdManager`.
```

Comment:
Description lists only have to `:`, so this line should be:

`SessionIdManager`::

It's typical to have them in their own line, and the description in a new line below it.